### PR TITLE
Tighter dls

### PR DESCRIPTION
**Pull request by ricardoom**
use this URL: http://localhost:4000/design-toolkit/discovery-item.html

The ask here is to make the information density tighter in lists, on Discovery bib items:
So what I did to make sure this is working properly is to pull down, npm install and run
https://github.com/NYPL-discovery/discovery-front-end/tree/linking-items-rg
(You'll need to npm install to get _this_ branch of the DTK to be used) the go to the URL below...

http://local.nypl.org:3001/research/collections/shared-collection-catalog/bib/b17482582

and then click on the View Full Record button below, and behold the information density.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (ktp242) [inline .vscode/settings.json]: Do we need this file?

**Comment by ktp242** [comment]
Works fine on my local machine. The `View Full` button's styles are not include in this PR. is it right?

File: .vscode/settings.json (hunk 1 above)
Comment (ricardoom) [reply .vscode/settings.json]: nope. adding to .gitignore.

**Comment by ricardoom** [comment]
@ktp242 correct, the button is not to be styled as its just a functionality placeholder.

File: .vscode/settings.json (hunk 1 above)
Comment (ktp242) [reply .vscode/settings.json]: If we don't need it, can we just delete the file?

Comment (ricardoom) [reply .vscode/settings.json]: should be ignored now...

Comment (EdwinGuzman) [reply .vscode/settings.json]: It's ignored but still in git. `git rm .vscode/settings.json`

**Comment by EdwinGuzman** [comment]
There's no spacing at all 😞 .
1) approving code here
2) discovery seems too crowded but I guess that's what was asked of you

**Pull request merged**